aws/aws-config.json

```diff
@@ -1,30 +1,29 @@
 {
     "mpc" : {
-        "command": "python -m honeybadgermpc.hbavss_multi",
+        "command": "python -m apps.shuffle.powermixing",
         "t": 1,
         "n": 4,
-        "k": 1000,
+        "k": 8,
         "port": 7000,
         "delta": -999,
         "num_triples": 1000,
         "skipPreProcessing" : true
     },
-    
+
     "aws":{
-        "credentials_file_path": "./aws/credentials.secret.json",
         "setup_file_path": "./aws/setup.sh",
 
         "region":{
-            "us-east-2": {
-                "vm_count": 5,
-                "security_group_ids": ["sg-f211d79b"],
-                "image_id": "ami-0f65671a86f061fcd",
-                "key_file_path": "./aws/rahul-us-east.pem",
-                "key_name": "rahul-us-east"
+            "us-east-1": {
+                "vm_count": 4,
+                "security_group_ids": ["sg-0faaf975"],
+                "image_id": "ami-0ac019f4fcb7cb7e6",
+                "key_file_path": "./aws/samarth-us-east1.pem",
+                "key_name": "samarth-us-east1"
```

Comment:
Now that we're working in `dev`, we should try to avoid this. AWS tests should use a per-user config, or even an AWS_KEY_NAME environment variable

Reply:
I will change these to generic values instead of my own and leave this file to be used only for reference.

Reply:
I have changed these to generic values. Since we support multiple geos having an environment variable won't work since we will need more than one.

I have left this config to be used for reference and any changes to it won't be checked in as I have added it to `.gitignore`.